storybook/cypress/e2e/header.cy.ts

```diff
@@ -56,15 +56,25 @@ describe("Header", () => {
   }
 
   it("should be accessible", () => {
+    prepareComponent();
+
     cy.checkA11y("dso-header");
-    // cy.percySnapshot()
-    cy.get("dso-header").invoke("attr", "useDropDownMenu", "always").checkA11y("dso-header");
-    cy
-      // .percySnapshot(`${Cypress.currentTest.title}" -- dropdown menu`)
-      .viewport(400, 600)
-      .get("dso-header")
-      .invoke("attr", "useDropDownMenu", "auto")
-      .checkA11y("dso-header");
+
+    cy.get("dso-header.hydrated")
+      .get("@dsoHeaderShadow")
+      .get("dso-icon.hydrated")
+      .then(() => {
+        cy.get("dso-header.hydrated").matchImageSnapshot();
+
+        cy.get("dso-header").invoke("attr", "useDropDownMenu", "always").checkA11y("dso-header");
+
+        cy.get("dso-header.hydrated")
+          .matchImageSnapshot(`${Cypress.currentTest.title} -- dropdown menu`)
+          .viewport(400, 600)
+          .get("dso-header")
+          .invoke("attr", "useDropDownMenu", "auto")
+          .checkA11y("dso-header");
+      });
```

Comment:
Er staat mij iets van bij dat je hebt gezegd dat Header een probleem component is. Is dat met deze code opgelost?

Reply:
Heb het aangeapst; zonder then-constructie

Reply:
En gaat Header goed?

Reply:
Ja, Header gaat nu goed.

Reply:
🤘